bolt-micronaut/src/main/java/com/slack/api/bolt/micronaut/SlackAppController.java

```diff
@@ -29,7 +28,8 @@ public SlackAppController(App slackApp, SlackAppMicronautAdapter adapter) {
     }
 
     @Post(value = "/events", consumes = {MediaType.APPLICATION_FORM_URLENCODED, MediaType.APPLICATION_JSON})
-    public HttpResponse<String> events(HttpRequest<String> request, @Body String body) throws Exception {
+    public HttpResponse<String> events(HttpRequest<String> request) throws Exception {
+        String body = request.getBody().orElse(null);
```

Comment:
issue in MN 4 - https://github.com/micronaut-projects/micronaut-core/issues/9694

Reply:
ok, thanks for the workaround